package.json

```diff
@@ -19,25 +19,22 @@
   ],
   "babel": {
     "presets": [
-      [
-        "@babel/preset-env",
-        {
-          "targets": {
-            "browsers": [
-              ">0.25%",
-              "not dead"
-            ]
-          }
-        }
-      ]
+      "@babel/preset-env",
+      "@babel/preset-react"
```

Comment:
Can we keep the targets?  

```
"targets": {
  "browsers": [">0.25%", "not dead"]
}
```

For our purposes it should match the fCC/fCC repo.

Reply:
To be honest, I just removed them, because I kept getting errors, after adding `preset-react`. So, I am not sure where they need to go.